app/controllers/api/v1/merchants_controller.rb

```diff
@@ -0,0 +1,33 @@
+# app/controllers/api/v1/merchants_controller.rb
+
+class Api::V1::MerchantsController < ApplicationController
+
+  def index
+    render json: MerchantSerializer.new(Merchant.all)
+  end
+
+  def show
+    render json: MerchantSerializer.new(Merchant.find(params[:id]))
+  end
+
+  def create
+    render json: MerchantSerializer.new(Merchant.create(merchant_params))
+  end
+
+  def update
+    render json: MerchantSerializer.new(Merchant.update(params[:id], merchant_params))
+  end
+
+  def destroy
```

Comment:
@BrianZanti @dionew1 Is there a way to verify the create/update/delete functionality in the local server (to ensure it matches the expectations in the requirements)? Or do we simply test for the presence of the expected attributes in our test suite?

Reply:
If you want to manually test this I would recommend looking into [Postman](https://www.postman.com/) to help format those requests.
I would also be sure to have a test for each of the requests.